src/SlowQuery.js

```diff
@@ -0,0 +1,35 @@
+import { performance } from 'perf_hooks';
+import logger from './logger';
+import rest from './rest';
+import auth from './Auth';
+
+export const registerSlowQueryListener = (app, options) => {
+  const slowQueryOptions = options.slowQuery;
+  app.use((req, res, next) => {
+    const startTime = performance.now();
+    res.on('finish', async () => {
+      const endTime = performance.now();
+      const duration = endTime - startTime;
+      const config = req.config;
+      if (duration > (options.slowQuery.threshold ?? 3000)) {
+        if (slowQueryOptions?.log) {
+          logger.warn(
+            `Detected a slow query on path ${req.path}. Duration: ${duration.toFixed(0)}ms`
+          );
+        }
+        try {
+          await rest.create(config, auth.master(config), '_SlowQuery', {
+            method: req.method,
+            path: req.path,
+            body: req.body,
+            query: req.query,
```

Comment:
Here i'm wondering about security, i can suggest some little improvements (inspired from Sentry):
- If body size is huge the _SlowQuery collection could be heavy, sentry currently limit the body size, may be we can splice the body at a defined size after a JSON.stringify (this size option could be added in the config).
- As i understand this hook will be triggered on all endpoints (create, update, get, find, login, signup etc...), here it's required to clean the body and remove sensitve data (like password, etc...), if i'm not wrong the `logger` already have this feature. May be a little package exists.